src/main/java/org/tron/common/runtime/RuntimeImpl.java

```diff
@@ -414,6 +480,11 @@ private void call()
     }
     byte[] code = this.deposit.getCode(contractAddress);
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Move the declaration of "code" closer to the code that uses it. ([squid:S1941](https://sonarcloud.io/organizations/tron-zhaohong/rules#rule_key=squid%3AS1941))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=java-tron&pullRequest=1670&issues=AWbSdHUuSgN4uCclEWnB&open=AWbSdHUuSgN4uCclEWnB)

[ISSUE_KEY]: AWbSdHUuSgN4uCclEWnB